packages/freezed/test/source_gen_src.dart

```diff
@@ -224,3 +224,30 @@ abstract class AstractClass {
 class _AstractClass implements AstractClass {
   const _AstractClass();
 }
+
+@ShouldThrow(
+  '@freezed classes configured with [finalize: true] must be sealed',
+)
+@Freezed(finalize: true)
+class FinalizedClassWithoutSealedKeyword {
+  FinalizedClassWithoutSealedKeyword._();
+  factory FinalizedClassWithoutSealedKeyword() =
+      _FinalizedClassWithoutSealedKeyword;
+}
+
+class _FinalizedClassWithoutSealedKeyword
+    implements FinalizedClassWithoutSealedKeyword {
+  _FinalizedClassWithoutSealedKeyword();
+}
+
+@ShouldThrow(
```

Comment:
Why this requirement?

Reply:
I expected that sealed classes must be extended rather than implemented, but it seems it doesn't matter, so it can be removed 👍

Reply:
Removed it